build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
         }
     }
 
-    ext.brainTreeVersion = "4.48.0"
+    ext.brainTreeVersion = "4.49.1"
```

Comment:
Do we typically call this out in the CHANGELOG?

Reply:
Oh yeah good call. I forgot to update the CHANGELOG actually.

Reply:
Added it here: [688b75a](https://github.com/braintree/braintree-android-drop-in/pull/479/commits/688b75ac5ec98ba68a19cc56dc8601acf440b977)